.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
Just a small note, for libraries that use universal config (i.e. "rc", "json", "js"), we prefer JS (because it has comment) or JSON (because vscode auto has schema for them). The "rc" one is quite limited

Reply:
Oh wait I'm wrong! VSCode recognize .rc file!